include/raft_types.h

```diff
@@ -20,6 +20,11 @@ typedef long int raft_term_t;
  */
 typedef long int raft_index_t;
 
+/**
+ * Id used to group entries into sessions
+ */
+typedef unsigned long raft_session_t;
```

Comment:
```suggestion
typedef unsigned long long raft_session_t;
```
Let's make it 64 bit at least